ha/monitor/k8s/object_monitor.py

```diff
@@ -167,3 +172,50 @@ def run(self):
                 break
         Log.info(f"Stopping the {self.name}...")
 
+    def _is_published_alert(self, alert) -> bool:
+        """
+        Check incoming alert is already published or not.
+        If incoming alert is not found in published alerts, then
+        it is a new alert to publish. Alert will be stored like,
```

Comment:
How many such alerts will be stored?

Reply:
It stores one alert per pod.
But there can be more alerts stored depends on how many pod created and deleted during k8s_monitor service run time.